### PR TITLE
Exclude java/util/WeakHashMap/GCDuringIteration.java

### PR DESCRIPTION
**Pull request by pshipton**
Issue https://github.com/eclipse/openj9/issues/10256

Signed-off-by: Peter Shipton <Peter_Shipton@ca.ibm.com>

### CODE REVIEW

**Comment by pshipton** [comment]
@smlambert

File: openjdk/ProblemList_openjdk11-openj9.txt [hunk 1]
```diff
@@ -261,6 +261,7 @@ java/util/stream/boottest/java.base/java/util/stream/NodeTest.java https://githu
 java/util/stream/test/org/openjdk/tests/java/util/SplittableRandomTest.java 	https://github.com/eclipse/openj9/issues/4613 	generic-all
 java/util/stream/test/org/openjdk/tests/java/util/stream/CountLargeTest.java	https://github.com/eclipse/openj9/issues/9040	linux-aarch64
 java/util/stream/test/org/openjdk/tests/java/util/stream/ToArrayOpTest.java	https://github.com/eclipse/openj9/issues/3447	generic-all
+java/util/WeakHashMap/GCDuringIteration.java	https://github.com/eclipse/openj9/issues/10256	generic-all
```
Comment (smlambert) [inline openjdk/ProblemList_openjdk11-openj9.txt:264]: Can you change the issue to point to https://github.com/AdoptOpenJDK/openjdk-tests/issues/1297 (a placeholder for permanent excludes, openjdk tests that are not and will never be applicable to openj9) for the 3 entries in this PR?  I will reference openj9/10256 in 1297.

When we add automation to reinclude tests by checking if issues in problemlists are closed, issue 1297 will be exempt, but other issues not.

Comment (pshipton) [reply openjdk/ProblemList_openjdk11-openj9.txt]: Updated.

**Review by smlambert — APPROVED** [review]

**Comment by smlambert** [comment]
Do not think our PR tests will verify this is excluded (as it currently is not a change-based check, related: https://github.com/AdoptOpenJDK/openjdk-tests/issues/2072#issuecomment-734385169), so merging ahead of the checks completing.

**Pull request merged**